modules/express/README.md

```diff
@@ -163,31 +163,48 @@ BitGo Express currently supports the following proxy protocols:
+
+```
+{
+"61f039aad587c2000745c687373e0fa9":"xprv9s21ZrQH143K3EuPWCBuqnWxydaQV6et9htQige4EswvcHKEzNmkVmwTwKoadyHzJYppuADB7Us7AbaNLToNvoFoSxuWqndQRYtnNy5DUY2",
+"61fb21819c54dd000755f8de3a18e46f":"xprv9s21ZrQH143K2tc1uz6E1rY1WYEaAQyFzY4C3qs4GMyX1uXy6YqdUXrWyKpm3AJmugPR56mC2ggJK5BsCenznJw2nGQ2P9AAEwZVLSVgjGr"
+}
+```
+
+
+![BitGo Express Signer Diagram](express_signer.png)
+
 ## Configuration Values
 
 BitGo Express is able to take configuration options from either command line arguments, or via environment variables.
 
-| Flag Short Name | Flag Long Name         | Environment Variable                     | Default Value | Description                                                                                                             |
-| --------------- | ---------------------- | ---------------------------------------- | ------------- | ----------------------------------------------------------------------------------------------------------------------- |
-| -p              | --port                 | `BITGO_PORT`                             | 3080          | Port which bitgo express should listen on.                                                                              |
-| -b              | --bind                 | `BITGO_BIND`                             | localhost     | Interface which bitgo express should listen on. To listen on all interfaces, this should be set to `0.0.0.0`.           |
-| -e              | --env                  | `BITGO_ENV`                              | test          | BitGo environment to interact with.                                                                                     |
-| -t              | --timeout              | `BITGO_TIMEOUT`                          | 305000        | Number of milliseconds to wait before requests made by `bitgo-express` time out.
-| -d              | --debug                | N/A, use `BITGO_DEBUG_NAMESPACE` instead | N/A           | Enable debug output for bitgo-express. This is equivalent to passing `--debugnamespace bitgo:express`.                  |
+| Flag Short Name | Flag Long Name         | Environment Variable                     | Default Value | Description                                                                                                                                                                                                                                                                                                                                         |
+| --------------- | ---------------------- | ---------------------------------------- | ------------- |-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
No need to add extra spaces to align these, they will automatically be spaced correctly in the rendered markdown.

Reply:
Ah okay, somehow my IDE did this automatically. I'll try to undo it.